inc/issue.class.php

```diff
@@ -526,29 +526,50 @@ public static function giveItem($itemtype, $option_id, $data, $num) {
       switch ("$table.$field") {
          case "glpi_plugin_formcreator_issues.name":
             $name = $data[$num][0]['name'];
-            return "<a href='".FORMCREATOR_ROOTDOC."/front/issue.form.php?id=".$id."&sub_itemtype=".$data['raw']['sub_itemtype']."'>$name</a>";
-            break;
+            $subItemtype = $data['raw']['sub_itemtype'];
+            switch ($subItemtype) {
+               case Ticket::class:
+                  $ticket = new Ticket();
+                  $ticket->getFromDB($id);
+                  $content = $ticket->fields['content'];
+                  break;
+
+               // TODO : need some code refactor to properly provide qtip
+               // case PluginFormcreatorFormAnswer::class:
+               //       $formAnswer = new PluginFormcreatorFormAnswer();
+               //       $formAnswer->getFromDB($id);
+               //       $content = $formAnswer->getFullForm();
+               //       // TODO : need to replace tags before creating the qtip
+               //       break;
```

Comment:
attempt to add qtip for form answers, but his led me to many code refactors. This refactor will be added for 2.8, maybe backported to 2.7.1 (not decided yet)